src/main/java/androidx/view/View.kt

```diff
@@ -84,6 +85,17 @@ inline fun View.doOnPreDraw(crossinline action: (view: View) -> Unit) {
     })
 }
 
+/**
+ * Sends {@link AccessibilityEvent} of type {@link AccessibilityEvent#TYPE_ANNOUNCEMENT}.
+ *
+ * @see View.announceForAccessibility
+ */
+@RequiresApi(16)
```

Comment:
I didn't use `Build.VERSION_CODES.JELLY_BEAN` to match the style of the other functions in this file.